src/Services/Air/AirParser.js

```diff
@@ -802,6 +802,14 @@ function airCancelPnr(obj) {
   throw new AirParsingError.CancelResponseNotFound();
 }
 
+function formSupplierLocatorBlock(supplierLocator) {
```

Comment:
Does this array ever have more than one item?
XML parser most of the time forms an array, where just single object has to be.

That's why we have some specific XML parsing settings in `src/Request/default-config.js`

Please check this result and the other parsed results too.
Please also attach getEmdItem and getEmdList JSON responses to this PR in comments

Reply:
Ok, checked this, it's not your code, and for this specific place it has to be an array. But I still would like to see methods output results